staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -433,11 +434,11 @@ func (o *GetOptions) transformRequests(req *rest.Request) {
 		return
 	}
 
-	group := metav1beta1.GroupName
-	version := metav1beta1.SchemeGroupVersion.Version
-
-	tableParam := fmt.Sprintf("application/json;as=Table;v=%s;g=%s, application/json", version, group)
-	req.SetHeader("Accept", tableParam)
+	req.SetHeader("Accept", strings.Join([]string{
+		fmt.Sprintf("application/json;as=Table;v=%s;g=%s", metav1.SchemeGroupVersion.Version, metav1.GroupName),
+		fmt.Sprintf("application/json;as=Table;v=%s;g=%s", metav1beta1.SchemeGroupVersion.Version, metav1beta1.GroupName),
+		"application/json",
+	}, ","))
```

Comment:
this prefers meta.k8s.io/v1

Reply:
How are you dealing with getting v1beta1 as a go struct in the code path?

Reply:
Table is type aliased, so type checks do not distinguish between them

Reply:
Nm, saw it.  Had a question there, ignore this.